configure

```diff
@@ -5,7 +5,7 @@
 check_cflags()
 {
 	check_cc_flag -std=gnu99 -pipe -Wall -Wshadow -Wcast-align -Wpointer-arith \
-		-Wwrite-strings -Wundef -Wmissing-prototypes -Wredundant-decls \
+		-Wwrite-strings -Wundef -Wmissing-prototypes \
```

Comment:
This causes a lot (and I mean a lot) of warnings on the OpenBSD version of the toolchain. I will move it back to an OpenBSD port patch.